pkgs/development/haskell-modules/configuration-darwin.nix

```diff
@@ -251,12 +251,17 @@ self: super: ({
   # Otherwise impure gcc is used, which is Apple's weird wrapper
   c2hsc = addTestToolDepends [ pkgs.gcc ] super.c2hsc;
 
-  # streamly depends on Cocoa starting with 0.8.0
-  streamly_0_8_1_1 = overrideCabal (drv: {
+  streamly = overrideCabal (drv: {
     libraryFrameworkDepends = [
       darwin.apple_sdk.frameworks.Cocoa
     ] ++ (drv.libraryFrameworkDepends or []);
-  }) super.streamly_0_8_1_1;
+  }) super.streamly;
+
+  streamly_0_8_2 = overrideCabal (drv: {
```

Comment:
I think you can just drop this, since it is unused and just bound to cause the same kind of regression.

Unfortunately, with the way `haskellPackages` is organized (versioned packages having `hydraPlatforms = [ ];`) and the jobset implemented (evaluating the package list (not the derivation) for all platforms on a single one), Hydra won't pick the removal of the attribute up for us.

Possibly this override should become a cabal2nix PostProcess rule.

Reply:
okay, i noticed that there's already a 
`  streamly = addBuildDepend darwin.apple_sdk.frameworks.Cocoa super.streamly;`